src/interpreter.jl

```diff
@@ -84,12 +84,19 @@ end
 
 # branch on https://github.com/JuliaLang/julia/pull/41328
 @static if isdefined(Compiler, :is_stmt_inline)
-function Compiler.inlining_policy(interp::CthulhuInterpreter, @nospecialize(src), stmt_flag::UInt8)
```

Comment:
The CIs are failing now because `1.8.0-DEV.472` doesn't seem to arrive at the nightly channel yet.
Still I'd like to drop the support for dev versions between https://github.com/JuliaLang/julia/pull/41328 and  `1.8.0-DEV.472`.